src/angular-pickadate.js

```diff
@@ -249,6 +249,7 @@
           defaultDate: '=',
           minDate: '=',
           maxDate: '=',
+          dayNames: '=?',
```

Comment:
hi there!

I would use `dayNames: '@?'` since we don't want to watch the `dayNames` attributes